ivy/functional/frontends/torch/nn/functional/loss_functions.py

```diff
@@ -333,6 +333,11 @@ def gaussian_nll_loss(
             "var has negative entry/entries"
         )
 
+    var = torch_frontend.tensor(ivy.array(var, copy=True))
+    ivy.set_with_grads(False)
+    var = torch_frontend.clamp(var, min=eps)
+    ivy.unset_with_grads()
```

Comment:
I don't understand why this grad handling was necessary. Have you tried using `ivy.maximum`?

Reply:
Hey I basically transpiled the code line by line from the torch implementation. The torch implementation for this piece of code was like this:

```
# Clamp for stability
var = var.clone()
with torch.no_grad():
    var.clamp_(min=eps)
```

Obviously, this might not be the most efficient way of implementing the function. But I just took the most simplest approach directly transpiring the above. If you have any specific direction I have to follow kindly let me know. Do you want me to remove the grad handling completely ?

Will something like this work ?

```
var = torch_frontend.clamp(var, min=eps)
```

or

```
var = ivy.maximum(ivy, min)
```


Also can you explain the necessity to use `ivy.maximum` in the place of `clamp`. I'm asking this to have better understanding of the framework and would help me to implement future functions easily.